Loop/Plugins/PluginManager.swift

```diff
@@ -10,20 +10,13 @@ import Foundation
 import LoopKit
 import LoopKitUI
 
-
-public struct AvailableDevice {
-    let identifier: String
-    let localizedTitle: String
-    let providesOnboarding: Bool
-}
-
-typealias AvailableService = AvailableDevice
-
 class PluginManager {
     private let pluginBundles: [Bundle]
+    private let extensionObjects: [NSObject]
```

Comment:
The first implementation of "extensions" didn't require them to be held onto.  Did that change?

Reply:
Yes, that changed. A "support" plugin is an extension that doesn't require a setup/settings UI, but changes the behavior of Loop. The object needs to stay around so it can be found later. See all of the "Support" related stuff.